<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainForm</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>80</y>
     <width>346</width>
     <height>321</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit#textEdit:focus{outline: none;
border:1px solid rgb(216, 216, 216);}
QTextEdit#textEdit{border:1px solid rgb(216, 216, 216);}</string>
   </property>
   <property name="lineWrapMode">
    <enum>QTextEdit::NoWrap</enum>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string>Source Code</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>430</y>
     <width>346</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextBrowser#textBrowser:focus{outline: none;
border:1px solid rgb(216, 216, 216);}
QTextBrowser#textBrowser{border:1px solid rgb(216, 216, 216);}</string>
   </property>
   <property name="placeholderText">
    <string>Analysis Infos</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="MarkTable">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>301</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>1</number>
   </property>
   <property name="columnCount">
    <number>3</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <row/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QWidget" name="TitleBar" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>690</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(68, 166, 150)</string>
   </property>
   <widget class="QPushButton" name="CloseBtn">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#CloseBtn{border:none}
QPushButton#CloseBtn:hover{border:none;background:rgb(72, 177, 160)}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/images/H:/Code/问答系统qt/shutdown.png</normaloff>:/images/H:/Code/问答系统qt/shutdown.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>13</width>
      <height>17</height>
     </size>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="TitleLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>7</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel#TitleLabel{color:white}</string>
    </property>
    <property name="text">
     <string>Language C Lexical Analyer</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="SourceCodeLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Source Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="MarkLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Word List</string>
   </property>
  </widget>
  <widget class="QLabel" name="InfoLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>407</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Analysis Infomation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>57</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
    </font>
   </property>
   <property name="text">
    <string>File Input</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
